kubernetes/services/ignite_config.xml

```diff
@@ -16,45 +15,55 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-
 <!--
     Configuration example with Kubernetes IP finder enabled.
 -->
 <beans xmlns="http://www.springframework.org/schema/beans"
-       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xsi:schemaLocation="
+	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Try to avoid space.

Reply:
Sure

Reply:
@kevin-zhonghao please check the ignite_config.xml. There are more spaces there.

Reply:
Fixed it